app/src/main/java/com/example/taskthree/viewmodel/CounterViewModel.kt

```diff
@@ -5,17 +5,15 @@ import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 
 class CounterViewModel : ViewModel() {
-    private var counter = MutableLiveData<Int>()
+    private val counter = MutableLiveData<Int>()
```

Comment:
primary constructor da counter'i 0 a ayarlamissin. Buna gerek olmadan 
```suggestion
    private val counter = MutableLiveData<Int>(0)
```
seklinde yapabiliriz. init{} bloguna bu durumda gerek kalmayacaktir